lib/engine/game/g_21_moon/game.rb

```diff
@@ -920,6 +920,12 @@ def sellable_bundles(player, corporation)
           @round.active_step.sellable_bundles(player, corporation)
         end
 
+        def close_corporation(corporation, quiet: false)
```

Comment:
why don't you just fix the base class?

Reply:
I don't know what that will break. I assume there was a reason it didn't have it. I'll investigate.

Reply:
See new commit.